chat-plugins/trivia.js

```diff
@@ -396,7 +396,7 @@ class Trivia extends Rooms.RoomGame {
 		// Trivia#destroy has already set the instance's room to null.
 		let tarRoom = this.room;
 		if (!tarRoom) {
-			for (let [roomid, room] of Rooms.rooms) { // eslint-disable-line no-unused-vars
+			for (const room of Rooms.rooms) { // eslint-disable-line no-unused-vars
```

Comment:
...wat

Reply:
Did you mean `Rooms.rooms.values()`?

Also you can remove the `eslint-disable-line`